manifest.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "Are men talking too much?",
+    "description": "Check who's dominating the conversation",
+    "start_url": "./",
+    "background-color": "#FFFFFF",
```

Comment:
shouldn't this be `background_color`? is there a good way to validate a manifest.json file for a website?